### ISSUE TITLE
Feature Request: Bypass Dex Login Screen - 1 to 1 mapping Connector to staticClient

### ISSUE THREAD

**Issue by nictrix** [dexidp/dex#1447]
Dex is super helpful in making all of our IDP upstreams generic and standardized.

I'd like an option to bypass the Dex login screen where it asks which provider to choose. Dex would automatically choose the provider based on the staticClient that sent the request for authentication.

This would make it a 1-to-1 mapping between staticCient and connector.

I've looked through the issues and found others that may have a similar request, unsure?
https://github.com/dexidp/dex/issues/1400
https://github.com/dexidp/dex/issues/32

Thanks for your time, please let me know what your thoughts are on this and if it's something you'd consider.

**Comment by srenatus** [comment]
#990 could also be relevant here, I suppose.

**Comment by nictrix** [comment]
Yeah, I think that is relevant, as it's asking for an automatic refresh to the connector it should already know about. My request is also making that possible on the initial auth flow.

**Comment by srenatus** [comment]
Ah, I see! I had missed that difference, sorry.
